### PR TITLE
feat(Kubernetes): Adding basic helm chart

### PR DESCRIPTION
**Pull request by hypery2k**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]: 
- [ ] I have read and understand the pull request rules.

# Description

Fixes #(issue)

## Type of change

Please delete any options that are not relevant.

- New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [ ] I have commented my code, particularly in hard-to-understand areas
  (including JSDoc for methods)
- [x] My changes generate no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Dcoumentation

I tested this chart for my deployment with the following values.yaml

```


ingress:
  enabled: true
  annotations:
    kubernetes.io/ingress.class: traefik
    kubernetes.io/tls-acme: "true"
    cert-manager.io/cluster-issuer: ca-issuer
  hostname: uptime....
  tls:
    secretName: uk-tls
resources: 
  limits:
     cpu: 500m
     memory: 512Mi
  requests:
     cpu: 300m
     memory: 256Mi
```

### CODE REVIEW

File: extra/helm/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: ws
+              containerPort: 3001
+              protocol: TCP
+          livenessProbe:
+            exec:
+              command:
+                - node
+                - extra/healthcheck.js
```
Comment (CommanderStorm) [inline extra/helm/templates/deployment.yaml:44]:
```suggestion
                - extra/healthcheck
```
See https://github.com/louislam/uptime-kuma/issues/3500

**Comment by louislam** [comment]
Thanks, but I think someone had made a helm chart already under the unofficial section. And usually I will not add any k8s support to the main branch. I did reject similar pull requests before: #2653 #935

https://github.com/louislam/uptime-kuma/wiki/%F0%9F%94%A7-How-to-Install#%EF%B8%8F-openshift-4-and-kubernetes-helm-3-chart-unofficial

**Comment by CommanderStorm** [comment]
> I did reject similar pull requests before

Should we remove the [`k8s-unofficial`](https://github.com/louislam/uptime-kuma/tree/k8s-unofficial) branch and remove this way of deployment from the docs? (I can provide a PR for the docs)

**Comment by louislam** [comment]
> > I did reject similar pull requests before
> 
> Should we remove the [`k8s-unofficial`](https://github.com/louislam/uptime-kuma/tree/k8s-unofficial) branch and remove this way of deployment from the docs? (I can provide a PR for the docs)

I think yes, because `k8s-unofficial` is unmaintained for a long time, I believe it is no longer working already.

And I just saw @k3rnelpan1c-dev is keep maintaining his repo, his helm should be the best.

**Comment by CommanderStorm** [comment]
> I think yes

Done.
@hypery2k Could you close this PR?

**Comment by hypery2k** [comment]
fine for me. Just overlooked the linked chart.

Thanks for the suggestion

**Pull request closed without merging**